### ISSUE TITLE
Not respecting *_DISABLE_ERRORS: false

### ISSUE THREAD

**Issue by omusavi** [nvuillam/mega-linter#361]
**Describe the bug**
Formatters are set to warnings by default. In order to turn warnings into errors, as I understand it, the appropriate `*_DISABLE_ERRORS` setting for each linter needs to be set to `false`. This does not currently work as it seems linters errors still show as warnings

**To Reproduce**

1. Create a .md file with a markdown linter error
2. In `.mega-linter.yml` set `MARKDOWN_MARKDOWNLINT_DISABLE_ERRORS: false`
3. Run mega-linter-runner

**Expected behavior**
Error code > 0 and linter shows an error for that markdown file

**Action behavior**
Markdown linter shows a warning state:
```
+----SUMMARY------+--------------------------+-------+-------+--------+--------------+
| Descriptor      | Linter                   | Files | Fixed | Errors | Elapsed time |
+-----------------+--------------------------+-------+-------+--------+--------------+
| ◬ MARKDOWN      | markdownlint             |     5 |       |      2 |         0.4s |
```

and the process also outputs:
```
◬ Successfully linted all files, but with ignored errors
```

**Screenshots**

**Additional context**
I have tried this with both:
```
MARKDOWN_MARKDOWN_TABLE_FORMATTER_DISABLE_ERRORS: false
MARKDOWN_MARKDOWNLINT_DISABLE_ERRORS: false
```

and in both cases where I expect errors, I see just warnings

**Comment by nvuillam** [comment]
Hmmm seems you're totally right, i'll have a look :)
Maybe add some other variable allowing to say that all formatters trigger errors, so you don't have to define a variable for each formatter ?

**Comment by omusavi** [comment]
Even better!

**Comment by nvuillam** [comment]
@omusavi tested, already available in **insiders** version, and currently being published into official version v4.3.0 :)
Have fun with `FORMATTERS_DISABLE_ERRORS: false` :)
And the bug you reported is also fixed :)

**Comment by omusavi** [comment]
Confirmed fix is working, thanks again!
